bpf/enhancedrecording/network.bpf.c

```diff
@@ -55,7 +59,16 @@ static int trace_connect_entry(struct sock *sk)
 static int trace_connect_return(int ret, short ipver)
 {
     u64 pid_tgid = bpf_get_current_pid_tgid();
-	u32 id = (u32)pid_tgid;
+    u64 cgroup = bpf_get_current_cgroup_id();
+	  u32 id = (u32)pid_tgid;
+	  u64 *is_monitored;
```

Comment:
nit: seems some mixed tabs and spaces here.

Reply:
Fixed